testapp/src/main/java/com/rakuten/tech/mobile/testapp/analytics/rat_wrapper/RATEvent.kt

```diff
@@ -28,28 +32,39 @@ class RATEvent {
         return action ?: ActionType.OPEN
     }
 
-    fun getLabel(): String {
-        return label ?: ""
+    fun getPgn():String{
+        return "$siteSection→$pageName"
```

Comment:
@munir-rakuten I don't think you need to append the `siteSection` to the page name. Page name and site section are sent as separate fields (`ssc` and `pgn`), so they don't need to be appended together.